transform/transformer.go

```diff
@@ -14,6 +14,11 @@ import (
 	"github.com/buger/jsonparser"
 )
 
+// JSONTransformer - a type implemented by the jstransform.Transformer and the transformers in content-schema
```

Comment:
nit: content-schema is a private repo, this is a public repo so for many people that comment won't quite make sense.